MeesGame/Gameplay/GameObjects/Player.cs

```diff
@@ -3,22 +3,24 @@
 using System;
 using System.Collections.Generic;
 using static MeesGame.PlayerAction;
+using Microsoft.Xna.Framework.Graphics;
 
 namespace MeesGame
 {
-    class Player : SpriteGameObject
+    class Player : GameObjectList
```

Comment:
Please explain this change carefully (the readme says that structure changes need to be mentioned, and I need a good reason why it can't be done with SpriteGameObject)